### PR TITLE
Add Polish README

### PR DESCRIPTION
**Pull request by ljedrz**
More compact variants of some expressions (e.g. "wyknać polecenie") might exist, but this should be an ok starting point.

### CODE REVIEW

File: README.pl.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+## Podpisy cyfrowe wydań
+
+Wydania są podpisywane przy pomocy klucza [B22DA89A](http://pgp.mit.edu/pks/lookup?op=vindex&search=0x52CB6609B22DA89A)
+i publikowane [na GitHubie](https://github.com/swaywm/sway/releases).
+
+## Instalacja
+
+### Z pakietów
+
+Sway jest dostępny w wielu dystybucjach Linuksa. Spróbuj zainstalować pakiet "sway" w swoim
```
Comment (maciejjo) [inline README.pl.md]: Sway is available on non-linux distros as well (e.g. FreeBSD), so and original document doesn't specify Linux so I'd omit that here

File: README.pl.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+_\*\*opcjonalnie: wymagane dla swaybg_
+
+Wykonaj następujące polecenia:
+
+    meson build
+    ninja -C build
+    sudo ninja -C build install
+
+Na systemach bez logind należy wyknać polecenie suid na binarce sway:
```
Comment (maciejjo) [inline README.pl.md]: s/wyknać/wykonać/

Comment (maciejjo) [inline README.pl.md]: I'd suggest not using 'binarka' as it is rather colloquial word - "plik binarny" would be more appropriate.

Comment (kreativka) [reply README.pl.md]: Maybe "pliku wykonywalnym" like an executable file?

File: README.pl.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+[Русский](https://github.com/swaywm/sway/blob/master/README.ru.md#sway--) - [Български](https://github.com/swaywm/sway/blob/master/README.bg.md#sway--) - [Español](https://github.com/swaywm/sway/blob/master/README.es.md#sway--) -
+[Polski](https://github.com/swaywm/sway/blob/master/README.pl.md#sway--)
+
+sway jest kompozytorem [Wayland](http://wayland.freedesktop.org/) kompatybilnym z i3.
+Przeczytaj [FAQ](https://github.com/swaywm/sway/wiki). Dołącz do [kanału IRC](http://webchat.freenode.net/?channels=sway&uio=d4)
+(#sway na irc.freenode.net).
+
+Jeśli chcesz wesprzeć rozwój sway, rozważ wsparcie SirCmpwn na jego [stronie Patreon](https://patreon.com/sircmpwn).
```
Comment (Wolf480pl) [inline README.pl.md]: Shouldn't `sway` be declinated, eg.
`Jeśli chcesz wsprzeć rozwój swaya,`
?

**Comment by ddevault** [comment]
Thanks!

File: README.pl.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+* meson \*
+* [wlroots](https://github.com/swaywm/wlroots)
+* wayland
+* wayland-protocols \*
+* pcre
+* json-c
+* pango
+* cairo
+* gdk-pixbuf2 \*\*
+* [scdoc](https://git.sr.ht/~sircmpwn/scdoc) >= 1.8.1 (opcjonalnie: strony pomocy man) \*
```
Comment (Wolf480pl) [inline README.pl.md]: `pomocy` or `podręcznika` ?

File: README.pl.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+_\*\*opcjonalnie: wymagane dla swaybg_
+
+Wykonaj następujące polecenia:
+
+    meson build
+    ninja -C build
+    sudo ninja -C build install
+
+Na systemach bez logind należy wykonać polecenie suid na pliku wykonywalnym sway:
```
Comment (Wolf480pl) [inline README.pl.md]: IMO `ustawić bit suid` - "set the suid bit", as opposed to "execute the suid command".

File: README.pl.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+## Konfiguracja
+
+Jeśli już korzystasz z i3, skopiuj swoją konfigurację i3 do katalogu `~/.config/sway/config` i
+zadziała od ręki. W przeciwnym razie skopiuj przykładowy plik konfiguracyjny do folderu
+`~/.config/sway/config`; zazwyczaj znajduje się w `/etc/sway/config`.
+Wykonaj polecenie `man 5 sway` aby uzyskać informacje dotyczące konfiguracji.
+
+## Uruchamianie
+
+Wykonaj polecenie `sway` z poziomu TTY. Niektóre menedżery wyświetlania mogą umożliwiać rozruch z ich
```
Comment (Wolf480pl) [inline README.pl.md]: Wouldn't `z TTYka` be more idiomatic?

Comment (Wolf480pl) [inline README.pl.md]: I'd rather translate it as `menadżery ekranu`, unless `menadżery wyświetlania` is an established term. Or as `display managery`, if we want to sacrifice grammar for clarity.

Comment (ljedrz) [reply README.pl.md]: Idiomatic, but colloquial (not unlike the aforementioned "binarka").

Comment (ljedrz) [reply README.pl.md]: I wasn't too sure about this one myself; I found this term used in the docs for Mint and openSUSE. "display managery" sounds a bit too informal for my taste.

File: README.pl.md (hunk 5 above)
Comment (ljedrz) [reply README.pl.md]: Good idea 👍; please feel free to adjust it (otherwise I can do it soon).

File: README.pl.md (hunk 4 above)
Comment (ljedrz) [reply README.pl.md]: Both seem equally rarely used if Google is to be trusted; I don't mind either.

File: README.pl.md (hunk 3 above)
Comment (ljedrz) [reply README.pl.md]: Probably a tad informal for the docs, but I have no strong feelings about this.

File: README.pl.md (hunk 6 above)
Comment (Wolf480pl) [reply README.pl.md]: The original English README is very informal (eg. "stop by the IRC channel"), but ok.

**Pull request merged**